Beta docs (#931)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaDirectoryRoleMember.md

# Get-EntraBetaDirectoryRoleMember

## Synopsis

Gets members of a directory role.

## Syntax

```powershell
Get-EntraBetaDirectoryRoleMember 
 -ObjectId <String> 
 [<CommonParameters>]
```

## Description

> The `Get-EntraBetaDirectoryRoleMember` cmdlet gets the members of a directory role in Microsoft Entra ID. Specify `ObjectId` to get members of directory role.

Reviewer: ```suggestion
The `Get-EntraBetaDirectoryRoleMember` cmdlet retrieves the members of a directory role in Microsoft Entra ID. To obtain the members of a specific directory role, specify the `ObjectId`. Use the `Get-EntraBetaDirectoryRole` cmdlet to get the `ObjectId` value.
```

Author: Changes done.

## Examples

### Example 1: Get members by role ID

```powershell
Connect-Entra -Scopes 'RoleManagement.Read.Directory'

> Get-EntraBetaDirectoryRoleMember -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"

Reviewer: ```suggestion
Get-EntraBetaDirectoryRoleMember -ObjectId '1708c380-4b8a-4977-a46e-6031676f6b41'
```

Author: Changes done.

```

```Output
Id                                   DeletedDateTime
--                                   ---------------

> bbbbbbbb-7777-8888-9999-cccccccccccc

Reviewer: ```suggestion
1708c380-4b8a-4977-a46e-6031676f6b41
```
